schema-templates/basic-auth.sql

```diff
@@ -0,0 +1,291 @@
+-------------------------------------------------------------------------------
+-- Adapted from https://github.com/robconery/pg-auth
+
+begin;
+
+create extension if not exists pgcrypto;
+create extension if not exists "uuid-ossp";
+create schema if not exists basic_auth;
```

Comment:
Why the new schema? Postgrest works only with one schema so you can't have your main tables in v1 schema and auth tables in basic_auth schema. I would say remove any schema reference all together and say in a comment something like "execute this sql in the schema where your tables live and make sure the names do not clash"